elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -151,8 +153,19 @@ protected void configure() {
                     }
                     bind(asyncAPIDao).to(AsyncAPIDAO.class);
 
-                    // TODO: If null, initialize with FileResultStorageEngine
                     ResultStorageEngine resultStorageEngine = asyncProperties.getResultStorageEngine();
+                    if (resultStorageEngine == null) {
```

Comment:
Why don't we move the initialization of ResultStorageEngine inside the if block for export enabled?